revapi-java/src/test/java/org/revapi/java/benchmarks/DifferenceTransformCodesBenchmark.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Revapi is Apache v2 licensed, I'm not sure we can mix the licenses like this. Is it ok for you to change it? 

I would also appreciate it if I were the copyright holder, if at all permissible from your side, to keep the licensing situation simple.